### DISCUSSION TITLE
Is it possible to embed tweet in post ?

### DISCUSSION THREAD

**Discussion by chouj** [Lednerb/bilberry-hugo-theme#563] [Q&A] [answered]
Hi. I'm a new user of this brilliant hugo theme. 

When I paste code generated by [Twitter Publish](https://publish.twitter.com/) for embeding a tweet in the post, it does not work. Hence,  just wondering if there is a shortcode for tweet embeding ? Found nothing after searching "twitter" in the repo except twitter card preview.

Look forward to reply. Thanks in advance.

**Comment by Sp-cy** [comment]
Hi. I'm not too fond of embedding tweets because of how impermanent the content in that site is. One day is gone and so the info that you were trying to portray with the embed. Have you tried the default hugo shortcode? It's only for individual posts and writing something like this should work: `{{% tweet 1682868651572203520 %}}`. If you want to use any of the other html embeds for the timeline you need to enable the unsafe markup config in your site configuration file.

**Reply by igor-baiborodine** [reply, marked as answer]
That's correct. Please use the raw HTML enabled. For more details, please see the documentation: https://github.com/Lednerb/bilberry-hugo-theme#raw-html.

**Reply by chouj** [reply]
Thanks guys. The default shortcode works after enable the raw HTML.

**Comment by igor-baiborodine** [comment]
@chouj, moving to the Discussions.
